src/Microsoft.AspNetCore.Mvc.RazorPages/DependencyInjection/RazorPagesOptionsExtensions.cs

```diff
@@ -43,7 +43,7 @@ public static RazorPagesOptions ConfigureFilter(this RazorPagesOptions options,
         /// <param name="path">The path of the Razor Page.</param>
         /// <param name="policy">The authorization policy.</param>
         /// <returns>The <see cref="RazorPagesOptions"/>.</returns>
-        public static RazorPagesOptions AuthorizePage(this RazorPagesOptions options, string path, string policy)
+        public static RazorPagesOptions AuthorizePage(this RazorPagesOptions options, string path, string policy = "")
```

Comment:
Here's a blog post that explains why it's a tricky to version these: http://haacked.com/archive/2010/08/10/versioning-issues-with-optional-arguments.aspx/